src/it/java/io/deephaven/benchmark/tests/standard/StandardTestRunner.java

```diff
@@ -213,20 +293,21 @@ void generateSourceTable() {
                 .add("int640", "int", "[1-640]")
                 .add("int1M", "int", "[1-1000000]")
                 .add("float5", "float", "[1-5]")
-                .add("str250", "string", "s[1-250]")
-                .add("str640", "string", "[1-640]s")
-                .add("str1M", "string", "v[1-1000000]s")
+                .add("str250", "string", "[1-250]")
+                .add("str640", "string", "[1-640]")
+                .add("str1M", "string", "[1-1000000]")
```

Comment:
These no longer have prefix / suffix, will that change the nature of tests against this data?

Reply:
The only reason I had the prefix/suffix (e.g. "s") to begin with was to make strings more obvious in a table.  But it looks like that's color-coded in DH anyway.